src/styles/components/overrides/_image-filters.scss

```
@@ -1,26 +1,174 @@
+}
+
+.degoog-img-sidebar-title {
+  font-size: $font-size-md;
+  font-weight: 500;
+  color: var(--text-primary);
+}
+
+.degoog-img-sidebar-close {
+  @include button-reset;
+  @include flex-center;
+  width: 2rem;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)